x-pack/metricbeat/module/airflow/statsd/data_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package statsd
+
+import (
+	"fmt"
+	"net"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/elastic/beats/v7/auditbeat/core"
+	_ "github.com/elastic/beats/v7/libbeat/processors/actions"
+	"github.com/elastic/beats/v7/metricbeat/mb"
+	mbtest "github.com/elastic/beats/v7/metricbeat/mb/testing"
+	_ "github.com/elastic/beats/v7/x-pack/metricbeat/module/statsd/server"
+)
+
+func init() {
+	mb.Registry.SetSecondarySource(mb.NewLightModulesSource("../../../module"))
+}
+
+const (
+	STATSD_HOST = "localhost"
+	STATSD_PORT = 8126
+)
+
+func getConfig() map[string]interface{} {
+	return map[string]interface{}{
+		"module":     "airflow",
+		"metricsets": []string{"statsd"},
+		"host":       STATSD_HOST,
+		"port":       STATSD_PORT,
+		"period":     "100ms",
```

Comment:
Period shouldn't be actually needed for this metricset :thinking:

Reply:
`statsd` has it. I've overridden it in the test to make it faster and be sure the ticker for reporting event will tick before listening for the push metric will timeout